openhantek/src/mainwindow.ui

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not so familiar with Qt restrictions; would be great to have entire .ui file stored in /res, not /src. There's no actual source code here, just layouts and action bindings.

Reply:
The ui files contain xml and they get converted to cpp code. They express sources in a different way like in a web project where you have html, javascript, css in a src directory.